src/lib/components/explorer/results/ResultsPanel.svelte

```diff
@@ -185,6 +185,9 @@
           {#each $filters as filter}
             <FilterComponent {filter} />
           {/each}
+          {#each $genomicFilters as filter}
+            <FilterComponent {filter} />
+          {/each}
```

Comment:
Not a huge fan of breaking out genomic filters. I think we will need all filters to be in the same list in the future. Its probably fine for MVP though.